pkgs/applications/editors/neovim/ruby_provider/gemset.nix

```diff
@@ -1,13 +1,23 @@
 {
+  logger = {
+    groups = ["default"];
+    platforms = [];
+    source = {
+      remotes = ["https://rubygems.org"];
+      sha256 = "0lwncq2rf8gm79g2rcnnyzs26ma1f4wnfjm6gs4zf2wlsdz5in9s";
+      type = "gem";
+    };
+    version = "1.6.1";
+  };
   msgpack = {
     groups = ["default"];
     platforms = [];
     source = {
       remotes = ["https://rubygems.org"];
-      sha256 = "1a5adcb7bwan09mqhj3wi9ib52hmdzmqg7q08pggn3adibyn5asr";
+      sha256 = "1a5fsgchkpcca4wf3pipbb2jbj523l7fbaq37j10cr0yymwlkc7z";
       type = "gem";
     };
-    version = "1.7.2";
+    version = "1.7.5";
```

Comment:
question: Is this update required for the logger? It looks unrelated.

Reply:
it is but it works? I can undo that part I guess but this is what bundix did

All I did was add `gem 'logger'` to the gemfile and then run this.

```nix
        default = pkgs.writeShellScriptBin "nvim_ruby_updater" ''
          echo "Running 'bundle update'..."
          ${rubyEnv}/bin/bundle update
          echo "Running 'bundix'..."
          ${pkgs.bundix}/bin/bundix
        '';
```

I would assume that the best msgpack version to use would be the one that bundix says but idk

Thats a generated file

Reply:
`nix run --no-write-lock-file github:BirdeeHub/neovim_ruby_updater`

I made a little util for updating it

Reply:
Im sure I could do that though but it doesnt seem to matter

Reply:
I don't think you need to rip it out, it was more about organizing commits into logical units to organize related changes. One was adding a dependency and one was a version bump of existing dependencies.

Reply:
Should they be separate PR or separate commits? I can do separate commits easy enough I guess?

Reply:
> Should they be separate PR or separate commits? I can do separate commits easy enough I guess?

Just separate commits to indicate one is adding a dependency to silence warnings and another to bump the version to latest release.

Reply:
honestly I dont know what bundled with does either

I kinda would rather just leave it as one because I know it works and I dont wanna break it

Reply:
I think if using `gem 'logger'` bumps another dependency, then it's fine to be in a single commit.

Reply:
> I dont know what happens if I manually downgrade it without changing bundled with is the issue XD

Bundled with looks like just metadata for explaining the version the command was run with.

Reply:
Does this really matter? I don't think it does much. Bundler bumped the package while regenerating the lock file which should be totally fine here.

Reply:
Never said that mattered, I don't know why it was hung up on.. lol

Reply:
Yeah it didnt seem important and it worked.

I figure because its a newer version of ruby in nixpkgs now the current version is best